packages/lib/src/toolbar/controls/ExportEntry.tsx

```diff
@@ -1,12 +1,18 @@
+import type { ForwardedRef, HTMLAttributes } from 'react';
+import { forwardRef } from 'react';
+
 import styles from './Selector/Selector.module.css';
 
 interface Props {
   format: string;
   url: URL | (() => Promise<URL | Blob>) | undefined;
 }
 
-function ExportEntry(props: Props) {
-  const { format, url } = props;
+const ExportEntry = forwardRef<
```

Comment:
Custom `render` components need to be [open for extension](https://ariakit.org/guide/composition#custom-components-must-be-open-for-extension) (i.e. forward `ref` and spread extra props).